backend/dal/Helpers/Extensions/BuildingExtensions.cs

```diff
@@ -14,18 +15,18 @@ namespace Pims.Dal.Helpers.Extensions
     public static class BuildingExtensions
     {
         /// <summary>
-        /// Make a query to determine if the building PID and PIN are unique.
-        /// - No two buildings should have the same PID (exception below)
-        /// - No two buildings should have the same PIN
-        /// - A Crown Land building without a Title will have a PID=0 and a unique PIN.
+        /// Make a query to determine if the building names are unique.
+        /// - No two buildings should have the same name on a parcel.
         /// </summary>
-        /// <param name="buildings"></param>
+        /// <param name="context"></param>
+        /// <param name="parcel"></param>
         /// <param name="building"></param>
-        /// <exception type="DbUpdateException">The PID and PIN must be unique.</exception>
-        public static void ThrowIfNotUnique(this DbSet<Entity.Building> buildings, Entity.Building building)
+        /// <exception type="DbUpdateException">The name within a parcel should be unique.</exception>
+        public static void ThrowIfNotUnique(this PimsContext context, Entity.Parcel parcel, Entity.Building building)
```

Comment:
This could possibly lead to confusion due to the fact buildings can now exist on multiple parcels.  A unique name may not be required anymore, but it generally doesn't make sense to have multiple buildings with the same name.

Reply:
Will have to consider whether or not to enforce this on the frontend as well.